packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-exec-start-conf

```diff
@@ -0,0 +1,8 @@
+[required-extensions]
+kubernetes = "v1"
++++
+[Service]
+{{#if settings.kubernetes.device-plugins.nvidia}}
```

Comment:
Did you test that this works correctly on instances without an nvidia device plugin set?

Reply:
Yes. I tested it if `settings.kubernetes.device-plugins.nvidia` is not defined then this file is rendered as 
```
[Service]
```
which are kind of ignored as there is no overriding settings. It works as expected.

Reply:
This feels weird and to someone just seeing this on the system would appear to be a failure? Can we add a comment in an else statement to say deliberately left this way or something? Is it not better to just remove the `[Service]` as well?